src/App.tsx

```diff
@@ -1,10 +1,47 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useMemo, useState } from 'react';
+import { getTodos } from './api/todos';
+import { Header } from './components/Header/Header';
+import { Footer } from './components/Footer/Footer';
+import { Todo } from './types/Todo';
 import { UserWarning } from './UserWarning';
+import { SortByStatus } from './types/SortByStatus';
+import { NotificationError } from
+  './components/NotificationError/NotificationError';
+import { TodoList } from './components/TodoList/TodoList';
 
-const USER_ID = 0;
+const USER_ID = 11196;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [hasError, setHasError] = useState<string | null>(null);
+  const [sortBy, setSortBy] = useState<SortByStatus>(SortByStatus.All);
+
+  useEffect(() => {
+    getTodos(USER_ID)
+      .then(setTodos)
+      .catch(() => {
+        setHasError('Unable to load a todo');
+      });
+  }, []);
+
+  const handleCloseError = () => {
+    setHasError(null);
+  };
+
+  const activeTodos = todos.filter(todo => !todo.completed).length;
```

Comment:
```suggestion
  const activeTodosNumber = todos.filter(todo => !todo.completed).length;
```
to describe better what is it